### ISSUE TITLE
Frontend validation should not validate dataModel in signing task

### ISSUE THREAD

**Issue by CelineTrammi** [Altinn/app-frontend-react#2091]
### Description of the bug

In signing task: When the user has sent in the form and wants to sign og reject the validation message comes up.
![image](https://github.com/Altinn/app-frontend-react/assets/61122289/37671f99-ce9a-42b4-a81c-b2c7427a797b)

Furthermore, the debugger shows the warning 
`When validating the datamodel, validator null returned a validation error without a field
{"severity":"warning","message":{"key":"DataElementValidatedAtWrongTask","params":["Task_1","Task_2"]},"category":0,"source":null}`

### Steps To Reproduce

Create app with v4 frontendt
Configure a signing task after the form task. 
After sending in the form, press either the "reject" or "sign" button.

**Comment by olemartinorg** [comment]
@CelineTrammi The fix for this was released in [v2.5.0](https://github.com/Altinn/app-frontend-react/releases/tag/v4.5.0) and should now be live (assuming you're using the latest v4 rolling release). Would you mind testing this to verify that the fix works? 🙏

**Comment by olemartinorg** [comment]
Update: More generic "validation during signing/confirm tasks" fixes were released back in v2.5.0, but this exact issue did not get a fix. It turns out that apps with signing enabled should always set their signature data elements to `minCount: 0`. This is because a signature element is added by the backend _when_ you click the button to proceed to the next task, but `minCount: 1` tells us that 'a minimum of 1 signature is needed _before_ the user is allowed to go the next task'. That validation is telling you (in a clunky way, yes) that you _can't proceed to sign until you've signed_. :facepalm:

TL;DR: Set this to `minCount: 0`.
